proto/ibc/applications/middleware/fee/v1/genesis.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+
+// GenesisState defines the fee middleware genesis state
+message GenesisState {
+  map<string, ibc.applications.middleware.fee.v1.Fee> packet_fees = 1;
```

Comment:
@AdityaSripal As you can see here its not possible to have a map with a struct as the key (or bytes) https://developers.google.com/protocol-buffers/docs/proto3#maps

I think we will need some sort of .String() fn on the PacketId regardless, in order to store this in a map. Wdyt?

Reply:
Also, what do you want to call the variable here? `packet_fees` seems wrong

Reply:
I don't think it needs to be explicitly a mapping. I don't think we do that in other genesis files either.

I would make a separate struct

```
GenesisFee{
  packetID    PacketID
  fee Fee
}

GenesisState{
  repeated GenesisFee
}
```

Reply:
```
  // An entry for the PacketsFees map
  message PacketsFeesMapEntry {
    ibc.core.channel.v1.PacketId           packet_id = 1;
    ibc.applications.middleware.fee.v1.Fee fee       = 2;
  }
  // A mapping of packets -> escrowed fees
  repeated PacketsFeesMapEntry packets_fees_map = 1;
```

I pushed this up before I saw your comment. Want me to change the naming?